app/views/agent/partials/NextPaymentPartial.scala.html

```diff
@@ -0,0 +1,56 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.helpers.DisplayDate
+@import java.time.LocalDate
+@import config.AppConfig
+
+@this(displayDate: DisplayDate)
+
+@(nextPaymentDate: Option[LocalDate],
+  payments: Int,
+  isOverdue: Boolean
+)(implicit messages: Messages, appConfig: AppConfig)
+
+@hasMultiple = @{ payments > 1  }
+
+@paragraphContent = @{
+  (nextPaymentDate, payments) match {
+    case (Some(date), 1) => displayDate(date)
+    case (Some(date), _) => messages("nextPaymentPartial.multiple", payments)
+    case _ => messages("nextPaymentPartial.noPayments")
+  }
+}
+
+<div class="tile tile-no-border">
+  <div id="next-payment" class="tile-body">
+    <h2 class="govuk-heading-m" id="next-payment-heading">
+      @if(hasMultiple) {
+        @messages("nextPaymentPartial.heading.multiple")
+      } else {
+        @messages("nextPaymentPartial.heading")
+      }
+    </h2>
+    <p class="govuk-body" id="next-payment-paragraph">
+      @paragraphContent
+      @if(isOverdue) { <strong id="overdue" class="task-overdue">@messages("nextPaymentPartial.overdue")</strong> }
```

Comment:
The overdue label should only be displayed when the client has one overdue payment. If they have more than one outstanding payment this shouldn't be displayed, regardless of whether any of the payments are overdue. Additionally, this seems to display even without any overdue payments (at least when I log in with vrn 888908544 as a principal user I only get one outstanding payment that is not overdue, but if I log in as an agent with that vrn I get the overdue label)